### ISSUE TITLE
updated translations and working download link

### ISSUE THREAD

**Issue by mtrz** [exebetche/vlsub#197]
In my forked VLSub version and updated some of the translated languages by the vlc translators.
I only included full translated languages with the latest strings in my fork.
I put also all languages together and put this file on pastebin, that can be reached also without SSL.
I also clean up the list of available languages to not to confuse the users, which languages are really supplied. You can use my version on: https://github.com/mtrz/vlsub
Or edit your own vlsub.lua and change on line 1081 the link there to:
http://pastebin.com/raw/nWs1yYfx

Background of the problem:
VLSub has only a very basic http parser and you can't just rename the links to https, your need SSL suport in the code, e.g. https links uses also a different port 443, not 80 as http, and many more...
You need additional libraries that enable SSL support with lua and deal with certificates and this feature would be very important for a correct working of vlsub in the future. Many pages switches completely to https and maybe also opensubtitles some day. If this won't be fixed, this could be the end of this very nice extension. I already try to implement it with different lua modules but without success at the moment. Maybe we have some lua genius here who can help.

Thanks,
Michi

**Comment by GorvGoyl** [comment]
is it working for english locale?

**Comment by mtrz** [comment]
Yes, on my forked version you will find an updated english locale file.
